__tests__/app.test.js

```diff
@@ -0,0 +1,53 @@
+const db = require("../db/connection");
+const app = require("../app");
+const testdata = require("../db/data/test-data/index");
+const seed = require("../db/seeds/seed");
+const request = require("supertest");
+
+afterAll(() => {
+  return db.end;
+});
+
+beforeEach(() => {
+  return seed(testdata);
+});
+
+xdescribe("GET /api/topics", () => {
+  test("GET respond with an array of topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        // console.log(res.body);
```

Comment:
Best to remove a console.log completely once you don't need it anymore, keeps things nice and clean!